### PR TITLE
OTT-1360: As OpenWrap user, disable dynamic fetch if floors module disabled from UI for all S2S profiles

### PR DESCRIPTION
**Pull request by pm-jaydeep-mohite**
# Description

Please add change description or link to ticket, docs, etc.

# Checklist:

- [ ] PR commit list is unique (rebase/pull with the origin branch to keep master clean).
- [ ] JIRA number is added in the PR title and the commit message.
- [ ] Updated the `header-bidding` repo with appropiate commit id.
- [ ] Documented the new changes.

For Prebid upgrade, refer: https://inside.pubmatic.com:8443/confluence/display/Products/Prebid-server+upgrade

### CODE REVIEW

**Review by Pubmatic-Supriya-Patil — APPROVED** [review]
LGTM

File: modules/pubmatic/openwrap/floors.go [hunk 1]
```diff
@@ -48,12 +39,15 @@ func setFloorsExt(requestExt *models.RequestExt, configMap map[int]map[string]st
 		}
 	}
 
-	url, urlExists := configMap[models.VersionLevelConfigID][models.PriceFloorURL]
-	if urlExists {
-		if requestExt.Prebid.Floors.Location == nil {
-			requestExt.Prebid.Floors.Location = new(openrtb_ext.PriceFloorEndpoint)
+	// Based on floorPriceModuleEnabled flag, dynamic fetch would be enabled/disabled
+	enableFlag, isFlagPresent := configMap[models.VersionLevelConfigID][models.FloorModuleEnabled]
```
Comment (pm-nikhil-vaidya) [inline modules/pubmatic/openwrap/floors.go:43]: Are we not setting enabled flag to false if floor is disabled from UI?

File: modules/pubmatic/openwrap/floors_test.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ func Test_setFloorsExt(t *testing.T) {
 		want *models.RequestExt
 	}{
 		{
-			name: "JSON URL is present in db",
```
Comment (pm-nikhil-vaidya) [inline modules/pubmatic/openwrap/floors_test.go:24]: Does other test cases did not afftected by this change? If we are missing some then can we add it

File: modules/pubmatic/openwrap/floors.go (hunk 1 above)
Comment (pm-jaydeep-mohite) [reply modules/pubmatic/openwrap/floors.go:43]: not setting enabled flag to false if floor is disabled from UI

File: modules/pubmatic/openwrap/floors_test.go (hunk 2 above)
Comment (pm-jaydeep-mohite) [reply modules/pubmatic/openwrap/floors_test.go:24]: Updated UT's

**Review by pm-nikhil-vaidya — APPROVED** [review]
LGTM

**Review by ashishshinde-pubm — APPROVED** [review]
LGTM

**Pull request merged**